app/channels/map_channel.rb

```diff
@@ -0,0 +1,8 @@
+class MapChannel < ApplicationCable::Channel
+  # Called when the consumer has successfully
+  # become a subscriber of this channel.
+  def subscribed
+    # TODO: verify permission
```

Comment:
I'm not precisely sure whether this is the appropriate place to determine whether the authenticated client has access to this stream of events related to a map. block people without permission to view the map from subscribing to events